fabric/colors.py

```diff
@@ -22,13 +22,21 @@
 If ``bold`` is set to ``True``, the ANSI flag for bolding will be flipped on
 for that particular invocation, which usually shows up as a bold or brighter
 version of the original color on most terminals.
+
+It is now possible to disable color printing completely by setting the environment
+variable FABRIC_DISABLE_COLORS.  The value doesn't matter as long as it exists.
 """
 
+import os
 
 def _wrap_with(code):
 
     def inner(text, bold=False):
         c = code
+
+        if os.environ.get("FABRIC_DISABLE_COLORS", None):
```

Comment:
This should probably check if `is None`, if the intention truly is to have any value for `FABRIC_DISABLE_COLORS` work. It's possible to set it to the empty string and this would then disable the colors. So either the documentation or the code should be updated to stay in sync.